### DISCUSSION TITLE
How to set clipPath with BackgroundImage

### DISCUSSION THREAD

**Discussion by MetesTakasaki** [fabricjs/fabric.js#8837] [Q&A]
Code -> [https://jsfiddle.net/yqopdf1e/60/](https://jsfiddle.net/yqopdf1e/60/)
Hi, I have a problem with the config **clipPath**. after I add text. text is not showing on boundingBox.

**Comment by iocoker** [comment]
You need your clipPath rectangle to have a fill, not transparent. Check -> [https://jsfiddle.net/xwk7rb6g/4/](https://jsfiddle.net/xwk7rb6g/4/)
